### ISSUE TITLE
Doesn't stop when password is found

### ISSUE THREAD

**Issue by daem0n707** [maxrooted/instashell#1]

**Comment by ghoxt007** [comment]
same issue, i guess it doesnt work

**Comment by arvishal26** [comment]
its only stop when the full list is done

**Comment by ghoxt007** [comment]
It doesn’t find password

**Comment by White-Hat007** [comment]
Exactly....It stops only when the whole list is checked. I don't think you can really call it as checking. Cause if that's the case it should've prompted me that it has found the right password. Lmao.
Right now, this script is useless....I'm just saying it in case someone is reading the issues before installing it.

**Comment by bhuveshrao198** [comment]
total waste of time .

**Comment by amir61v** [comment]
It's totally useless. i gave it a real account and it's password and it didn't find it. maybe it used to work before but now it needs some recheking. thanks for writing it anyway.

**Comment by eeblocks** [comment]
The same happens to me, what a horror ..

**Comment by anshuman953** [comment]
Me too i literally created a wordlist filled with just my password 100 times in a row, it still didnt catch anything

**Comment by yashgarg7** [comment]
same problem...can anyone tell the solution for this

**Comment by Golden185** [comment]
> Exactly....It stops only when the whole list is checked. I don't think you can really call it as checking. Cause if that's the case it should've prompted me that it has found the right password. Lmao.
> Right now, this script is useless....I'm just saying it in case someone is reading the issues before installing it.

You need to stay 9-11 h to examinate all passwords!

**Comment by blibli88** [comment]
My password research doesn't even begin.
In the () , at the end of the password list file name, no number appears (that it would be the number of words in the password list).
Why? (i'm using unix shell on osx)
